css/daccs-css.css

```diff
@@ -1,3 +1,171 @@
+/* For mobile phones: */
+[class*="w-"] {
```

Comment:
For this particular one it's because the width size will be good for one screen size but bad for another.  If they were static css classes the one added later in the html tag will override the one added earlier.

Reply:
I understand but I think that this is even more of a reason why a strategy like the one proposed here (https://github.com/DACCS-Climate/daccs-website/pull/11#discussion_r1272428188) is a better approach. That way we can avoid using these sort of dynamic lookups entirely

Reply:
Also, do you want to be matching `w-` anywhere in the string or just at the beginning? If just at the beginning then you probably want the `^=` selector